generators/app/templates/_.eslintrc

```diff
@@ -1,54 +1,85 @@
 {
-	"env": {
-		"browser": true,
-		"node": true
-	},
-	"parserOptions": {
-		"ecmaVersion": 2019
-	},
-	"globals": {
-		"sap": true,
-		"jQuery": true
-	},
-	"rules": {
-		"block-scoped-var": 1,
-		"keyword-spacing": 2,
-		"space-unary-ops": 2,
-		"camelcase": 1,
-		"consistent-return": 1,
-		"no-warning-comments": 1,
-		"default-case": 1,
-		"no-console": 2,
-		"no-unused-vars": 2,
-		"no-debugger": 2,
-		"no-trailing-spaces": [
-			2,
-			{
-				"ignoreComments": true
-			}
-		],
-		"semi": [
-			1,
-			"always"
-		],
-		"quotes": [
-			1,
-			"double"
-		],
-		"key-spacing": [
-			1,
-			{
-				"beforeColon": false
-			}
-		],
-		"comma-spacing": [
-			1,
-			{
-				"before": false,
-				"after": true
-			}
-		],
-		"no-shadow": 2,
-		"no-irregular-whitespace": 2
-	}
-}
\ No newline at end of file
+    "env": {
+        "browser": true,
+        "node": true,
+        "es6": true,
+        "jquery": true
+    },
+    "parserOptions": {
+        "ecmaVersion": 2020
+    },
+    "globals": {
+        "sap": true,
+        "jQuery": true
+    },
+    "rules": {
+        "array-callback-return": "error",
+        "arrow-spacing": "error",
+        "brace-style": "error",
+        "camelcase": ["warn"],
+        "comma-spacing": [
+            "warn",
+            {
+                "before": false,
+                "after": true
+            }
+        ],
+        "comma-style": "error",
+        "consistent-return": "warn",
+        "default-case": "warn",
+        "default-param-last": "error",
+        "eqeqeq": "error",
+        "id-length": "off",
+        "key-spacing": [
+            "warn",
+            {
+                "beforeColon": false
+            }
+        ],
+        "keyword-spacing": "error",
+        "max-lines": [
```

Comment:
Hmm, this is indeed a good practise to reduce the size/length of the modules - sometimes this isn't easily possible without syntetic splits (big controllers) - but as it is a warning, I am fine with it.

Reply:
And I think, if some one really runs into (project-specific) issues they can always change the setting for this project. But in general the developer should be notified that he reaches some limit.

Reply:
Agreed 👍